roxie/ccd/ccdfile.cpp

```diff
@@ -288,7 +288,11 @@ class CRoxieLazyFileIO : implements ILazyFileIO, implements IDelayedFile, public
                         if (isCompressed && !isKey)
                             current.setown(createCompressedFileReader(f));
                         else
+                        {
                             current.setown(f->open(IFOread));
+                            if (blockedIndexIOSize)
```

Comment:
that's true, mainly because I thought use case tends to be different in roxie, where unlikely to have some indexes on a fast/non-blob plane and others on a slow/blob plane.. - and therefore a per plane configuration setting would not be required as much.

Reply:
now added